.travis.yml

```diff
@@ -13,13 +13,14 @@ services:
   - docker
 
 env:
-  - DEX_POSTGRES_DATABASE=postgres DEX_POSTGRES_USER=postgres DEX_POSTGRES_HOST="localhost" DEX_ETCD_ENDPOINTS=http://localhost:2379 DEX_LDAP_TESTS=1 DEBIAN_FRONTEND=noninteractive
+  - DEX_POSTGRES_DATABASE=postgres DEX_POSTGRES_USER=postgres DEX_POSTGRES_HOST="localhost" DEX_ETCD_ENDPOINTS=http://localhost:2379 DEX_LDAP_TESTS=1 DEBIAN_FRONTEND=noninteractive DEX_KEYSTONE_URL=http://localhost:5000 DEX_KEYSTONE_ADMIN_URL=http://localhost:35357
 
 install:
   - sudo -E apt-get install -y --force-yes slapd time ldap-utils
   - sudo /etc/init.d/slapd stop
   - docker run -d --net=host gcr.io/etcd-development/etcd:v3.2.9
-
+  - docker run -d -p 0.0.0.0:5000:5000 -p 0.0.0.0:35357:35357 openio/openstack-keystone
```

Comment:
❓ Is there any version pins we could add? Right now, this will take the "latest" tag, and that'll potentially break a month (or a year) from now, and will cause some PR to go ":x:" for the wrong reasons :wink:

Reply:
This image has only "latest" tag, so the only thing we can do here is to fork it on dockerhub and add specific tag.

Reply:
👍 Let's be optimistic. We can still fork it if the need arises down the road. (But really, I've rather avoid that 😅)

Reply:
Is there an official image for Keystone? I don't trust a "latest" tag from a organization that doesn't maintain the project.

Reply:
I could not find an "official" image for Keystone. I've asked the maintainer of the image to add tag, so right now we are using "pike" tag. Is this good enough ?

Reply:
I think it's a good compromise. Thanks for engaging with the maintainer there 👍